reactor-netty-http/src/test/java/reactor/netty/http/client/HttpRedirectTest.java

```diff
@@ -593,23 +527,19 @@ void testLastLocationSetToResourceUrlOnRedirect() throws CertificateException {
 		SslContextBuilder serverSslCtxBuilder =
 				SslContextBuilder.forServer(cert.certificate(), cert.privateKey());
 		DisposableServer redirectServer =
-				HttpServer.create()
-				          .port(0)
+				createServer()
```

Comment:
same here with `redirectServer` and `initialServer`

Reply:
fixed